contracts/RMRK/extension/revealable/IRMRKRevealer.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.21;
+
+interface IRMRKRevealer {
+    /**
+     * @notice Returns the assetIds to reveal for the given tokenIds
+     * @param tokenIds The tokenIds to reveal
+     * @dev This method MUST only return existing assetIds
+     * @dev This method MUST return the same amount of assetIds as tokenIds
+     * @return revealedAssetIds The assetIds to reveal
+     * @return assetToReplaceIds The assetIds to replace
+     */
+    function getRevealedAssets(
```

Comment:
The name of the function sounds like the assets were already revealed, but comment says `assetIds to reveal` like they are yet to be revealed

Reply:
I would rename it to `getRevealableAssets` and `revealableAssetIds` if this is about assets that are yet to be revealed

Reply:
Yes I like it more, but then we have an opposite issue, that once they are revealed, the name will not be very accurate, as they are already revealed, but var is still called `toReveal`. Another alternative is `assetIdsOnReveal` or something like that. But if you don't like any of these options, you can leave as is

Reply:
Since this became reveal instead, I think `revealedAssetsIds` is now fine